frontend/src/components/SideNav.test.tsx

```diff
@@ -133,6 +133,16 @@ describe('SideNav', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('renders jobs as active page', () => {
+    tree = shallow(<SideNav page={RoutePage.JOBS} {...defaultProps} />);
+    expect(tree).toMatchSnapshot();
```

Comment:
please change all `toMatchSnapshot()` to `toMatchInelineSnapshot()` per https://github.com/kubeflow/pipelines/issues/5118#issuecomment-778553197

If the snapshot is too long and not readable, consider:
* use snapshot-diff to make it shorter
* or whether the test should be a snapshot test or not, we can assert specific nodes in the tree directly too

(Note that, even if we use standalone snapshot files, they are also not readable. Using inline snapshots only reveal the problem rather than cause the problem.)

Reply:
@Bobgy got it!